apps/parsley/src/context/LogContext/index.tsx

```diff
@@ -227,19 +229,20 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
     });
 
     return results;
-  }, [
-    state.searchState.searchTerm,
-    processedLogLines,
-    upperRange,
-    lowerRange,
-    getLine,
-    dispatch,
-  ]);
+    // processedLogLines doesn't need to be in the dependency array because the search results would
+    // only change if a filter is applied -- and a filter and search term can't be applied at the same time.
```

Comment:
comment might be misleading because you can apply a filter and then apply a search on top of that at the same time